### ISSUE TITLE
Markdown render Issue

### ISSUE THREAD

**Issue by a7hybnj2** [inventree/InvenTree#662]
I was using the 'Notes' tab within a part today and the saved output doesn't render the markdown correctly.

```
If you have a line with 2 spaces at the end  
Then the next line should be rendered as a CR  
If there aren't spaces then it should...
be the same line
```

If you add that to a note you can see the issue. The live render appears correct but the saved render doesn't.

**Comment by SchrodingersGat** [comment]
This seems like it might be a limitation of [django-markdownify](https://github.com/erwinmatijsen/django-markdownify) which is the library I am using for markdown rendering.

**Comment by a7hybnj2** [comment]
Well if thats the case we can push this issue to them.

I will look at markdownify and check some things before creating issue.

Mar 8, 2020, 08:04 by notifications@github.com:

**Comment by a7hybnj2** [comment]
@SchrodingersGat I was looking at this again and it really confuses me. Can you take another look.

Below the rendering is being done correctly. So, whatever options are set in this live preview mode is perfect.
<img width="1741" alt="Screen Shot 2020-04-27 at 9 34 44 PM" src="https://user-images.githubusercontent.com/36831279/80437206-04e5d380-88cf-11ea-9691-9de6bdd4f8a6.png">

But when you save the rendering is lost. Are there different options used for the saved data rendering? Could you be stripping off trailing spaces in some kind of string fixer or sanitation?
<img width="1739" alt="Screen Shot 2020-04-27 at 9 34 52 PM" src="https://user-images.githubusercontent.com/36831279/80437205-044d3d00-88cf-11ea-8f13-527f1e7e2c14.png">

**Comment by a7hybnj2** [comment]
I did something! I found `settings.py` setting `MARKDOWNIFY_BLEACH` and changed it to false. Looks like it was stripping its own markdown formatting. I don't see a reason if you are using markdown that you would ever want to bleach it.

Thoughts?
<img width="1743" alt="Screen Shot 2020-04-27 at 9 41 59 PM" src="https://user-images.githubusercontent.com/36831279/80437596-f77d1900-88cf-11ea-86bc-2bf1a898dc5f.png">
